lib/msal-browser/rollup.config.js

```diff
@@ -0,0 +1,76 @@
+import typescript from "rollup-plugin-typescript2";
+import { uglify } from "rollup-plugin-uglify";
+import resolve from "rollup-plugin-node-resolve";
+import pkg from "./package.json";
+
+const libraryHeader = `/*! ${pkg.name} v${pkg.version} ${new Date().toISOString().split('T')[0]} */`;
+const useStrictHeader = `'use strict';`;
+const fileHeader = `${libraryHeader}\n${useStrictHeader}`;
+
+export default [
```

Comment:
would like to understand why you chose roll up and the comparison to webpack.
Also would like to hear plan if we choose this to standardize.

do we want to make a common package ? 

/lib/rollup/webpack-config 

that standardizes the common things and takes the rest as params

```js 
import { createRollupApp } from 'rollup-msal';

export default createRollupApp('msal-browser', {  sourceMaps: true, someOtherOption: true }) 

```

Reply:
In the main PR description I added some comparisons between the two and the advantages rollup has over webpack that make it a better choice for us.

Reply:
I think each msal package should have its own rollup config file. This will enable us to use different build definitions depending on the platform.